Src/PerformanceCollector/Shared/Implementation/QuickPulse/QuickPulseTimeProvider.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.Implementation.QuickPulse
+{
+    using System;
+
+    internal class QuickPulseTimeProvider
```

Comment:
Can you pls call it IClock for consistency with https://github.com/Microsoft/ApplicationInsights-dotnet/blob/271edd5fd8b9c7d84a9bd72448ec14f9b2145259/src/Core/Managed/Shared/Extensibility/Implementation/IClock.cs#L5